src/font.ts

```diff
@@ -316,31 +278,4 @@ export class Font {
     document.fonts.add(fontFace);
     return fontFace;
   }
-
-  /**
-   * Load the web fonts that are used by your app.
-   * If fontNames is undefined, all fonts in Font.WEB_FONT_FILES will be loaded.
-   *
-   * For example, `flow.html` calls:
-   *   `await Vex.Flow.Font.loadWebFonts();`
-   * Alternatively, you may load web fonts with a stylesheet link (e.g., from Google Fonts),
-   * and a @font-face { font-family: ... } rule in your CSS.
-   *
-   * You can customize `Font.WEB_FONT_HOST` and `Font.WEB_FONT_FILES` to load different fonts
-   * for your app.
-   */
-  static async loadWebFonts(fontNames?: string[]): Promise<void> {
-    const allFiles = Font.WEB_FONT_FILES;
-    if (!fontNames) {
-      fontNames = Object.keys(allFiles);
-    }
-
-    const host = Font.WEB_FONT_HOST;
-    for (const fontName of fontNames) {
-      const fontPath = allFiles[fontName];
-      if (fontPath) {
-        Font.loadWebFont(fontName, host + fontPath);
-      }
-    }
-  }
```

Comment:
Same as above. I don't see why we need to remove the ability to loadWebFonts() from a CDN.

I think having an easy API to load a desired font is important. Otherwise, a developer will have to look through our documents to understand how to deploy their own fonts on their own preferred CDN.

Reply:
reverted